app/views/users/show.html.erb

```diff
@@ -24,7 +24,7 @@
 </table>
 
 <% if user_signed_in? && current_user.id == @user.id %>
-  <%= render 'status_form' %>
+  <%= render 'home/status_dropdown' %>
```

Comment:
Maybe that file should be placed in another position when it is not exclusive to home. Like "common" for example.